### ISSUE TITLE
Unable to login with Debian based distros

### ISSUE THREAD

**Issue by unnilennium** [Sandmann79/xbmc#420]
**Addon used**
- [ ] Amazon (plugin.video.amazon)
- [x] Amazon VOD (plugin.video.amazon-test)
- [ ] Browser Launcher (plugin.program.browser.launcher)
- Addon Version [e.g. 0.7.5]

**Prime Video Account (select one)**
- [ ] primevideo.com
- [x ] amazon.ca

**System Setup (please provide the following information):**
- Hardware:  RPi 4 
- OS Version: raspbian buster up to date
- Kodi Version 18.5

**Upload Logs**
```
2020-03-05 12:18:29.048 T:2752307408  NOTICE: [Amazon VOD] Switching UserAgent
2020-03-05 12:18:29.049 T:2752307408  NOTICE: [Amazon VOD] Loading list of common UserAgents
2020-03-05 12:18:29.052 T:2752307408  NOTICE: [Amazon VOD] getURL: http://www.skydubh.com/pub/useragents.json
2020-03-05 12:18:30.401 T:2752307408  NOTICE: [Amazon VOD] Using UserAgent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36
2020-03-05 12:18:30.415 T:2752307408  NOTICE: [Amazon VOD] Login
2020-03-05 12:18:30.429 T:2752307408  NOTICE: [Amazon VOD] Retrieve territoral config
2020-03-05 12:18:30.431 T:2752307408  NOTICE: [Amazon VOD] getURL: https://na.api.amazonvideo.com/cdp/usage/v2/GetAppStartupConfig?deviceTypeID=A28RQHJKHM2A2W&deviceID=6b8fdb1588f89f0a9244500bb7ac5b2355f2b06d94bce98038f5425f&firmware=1&ve
2020-03-05 12:18:30.936 T:2752307408   ERROR: [Amazon VOD] Error reason: HTTPSConnectionPool(host='na.api.amazonvideo.com', port=443): Max retries exceeded with url: /cdp/usage/v2/GetAppStartupConfig?deviceTypeID=A28RQHJKHM2A2W&deviceID=6
2020-03-05 12:18:31.083 T:2752307408 WARNING: CPythonInvoker(8, /home/pi/.kodi/addons/plugin.video.amazon-test/default.py): the python script "/home/pi/.kodi/addons/plugin.video.amazon-test/default.py" has left several classes in memory t
```

Upload `kodi.log` somewhere and post the link here. If you need help finding it, [read the appropriate entry in the FAQs](https://github.com/Sandmann79/xbmc/wiki/FAQ#what-is-debug-logging--how-can-i-enable-debug-logging-in-kodi).
For Login issues, enable advanced logging in the settings, upload the `(amazon/avod)-login` files from the [Kodi Home](https://kodi.wiki/view/Special_protocol) folder, and paste the link here, as well.
 ==> will send more comprehensive bug 

**Describe the bug**
trying to login I never got the login page

also the dependency show me that there ia a missing dependency win_inet_pton
trying to install it with zip fails each time 

tryied to install it using pip, it is installed but still shown as missing

sudo python -m pip install win_inet_pton

**Comment by Varstahl** [comment]
95% probability the problem is the changes in debian. na.api.amazonvideo.com is a TLS1.0 only endpoint, debian supports strictly TLS1.2+. You need to alter the configuration for the time it takes to login. [Check this comment](https://github.com/Sandmann79/xbmc/issues/326#issuecomment-546262203) for information on how to do that.

If you already did that, enable [debug logging](https://github.com/Sandmann79/xbmc/wiki/FAQ#what-is-debug-logging--how-can-i-enable-debug-logging-in-kodi) and attach kodi.log file here, or use the uploads methods built in Kodi and post the links here.

**Comment by unnilennium** [comment]
that did the trick

may I suggest adding this to the README.md in order to get the information before opening a new issue, as it is not obvious to seek in closed issues for something occurring in last release ?

thanks

**Comment by Varstahl** [comment]
Technically this solution sucks, and as someone who has at least TLS1.2 or higher on his servers, I wanted to find for a solution to this ridiculous situation, although it's on amazon's and not us directly. Since it's a solution already available in the issue that keeps duplicating I'll just update this thread and keep it open or pin it.

**Comment by Sandmann79** [comment]
Fixed with 04695f5afb7ac82e1721b34f64db5720d31df6c1 and 29851fcfb98544c3deca7946b952df34ea3b0ec9.
